docs/veramo_agent/cli_tool.md

```diff
@@ -26,7 +26,7 @@ To check the CLI has installed, run:
 veramo -v
 
 # Output
-2.x.x
+3.x.x
```

Comment:
question: I wonder if there's a better automated way to get current version in Docusaurus...

Reply:
I'm sure there is a better way, but we didn't want to overengineer something without needing it first.

As our documentation base grows, some more automation will be needed.

Please share if you have ideas :)